src/hssm/likelihoods/blackbox.py

```diff
@@ -0,0 +1,87 @@
+"""Black box likelihoods written in Cython for "ddm" and "ddm_sdv" models."""
+
+from __future__ import annotations
+
+import hddm_wfpt
+import numpy as np
+
+
+def logp_ddm_bbox(data: np.ndarray, v, a, z, t) -> np.ndarray:
+    """Compute blackbox log-likelihoods for ddm models."""
+    x = (data[:, 0] * data[:, 1]).astype(np.float64)
```

Comment:
This step assume -1, 1 coding for choices. (negative rts correspond to -1 choices)
For 0, 1 coding this will break likelihood evaluation.
Do we enforce that actually?

Reply:
Maybe we want to enforce that? What was the assumption in HDDM?

Reply:
It's more that the LAN likelihoods specifically do not enforce this and operate under the assumption that decisions are [0, 1] (if I am not gravely misremembering). 

On the LAN side this is important because that's what the networks are trained on.

For consistency, I think we should stick with `[0, 1]` and just apply the change to `[-1, 1]` inside the blackbox likelihood wrapper for the c-functions. 

We can also allow both on the user-facing side, check for it and apply necessary changes before likelihood construction, but it's probably better to give users strict / simple advice on what to do (supply choices as [0, 1] for 2-choice models) and then stick with it.

Reply:
To answer the question directly: In `hddm` the guidance was [-1, 1] :).

Reply:
Okay I changed this behavior. Now it can deal with both assumptions :)